contracts/number/types/UFixed18.sol

```diff
@@ -104,6 +104,20 @@ library UFixed18Lib {
         return UFixed18.wrap(UFixed18.unwrap(a) * BASE / UFixed18.unwrap(b));
     }
 
+    /**
+     * @notice Divides unsigned fixed-decimal `a` by `b`
+     * @param a Unsigned fixed-decimal to divide
+     * @param b Unsigned fixed-decimal to divide by
+     * @param roundUp Whether to round result up to the next integer
+     * @return Resulting divided unsigned fixed-decimal
+     */
+    function div(UFixed18 a, UFixed18 b, bool roundUp) internal pure returns (UFixed18) {
```

Comment:
since we're kind of doing the overload backwards here, what if instead of the flag we just have two functions: `div` and `divOut`?

Reply:
Sure, I assume same goes for `Fixed18` as well?